src/core.js

```diff
@@ -33,6 +33,26 @@ function setPresentU(value, history) {
   )
 }
 
+function setPresentUMut(value, history) {
```

Comment:
```suggestion
function setPresentReplaceU(value, history) {
```

* The word `replace` is used in the configuration passed to [`init`](https://github.com/calmm-js/partial.lenses.history?tab=readme-ov-file#H-init).
* The weird naming convention here is that the suffix `U` is used for uncurried functions.